testpar/t_subfiling_vfd.c

```diff
@@ -427,6 +427,9 @@ test_stripe_sizes(void)
     dxpl_id = H5Pcreate(H5P_DATASET_XFER);
     VRFY((dxpl_id >= 0), "DCPL creation succeeded");
 
+    /* Set selection I/O mode on DXPL */
+    VRFY((H5Pset_selection_io(dxpl_id, H5D_SELECTION_IO_MODE_ON) >= 0), "H5Pset_selection_io succeeded");
```

Comment:
There is a minor inconsistency in the messages.  Sometimes, "<function name> succeeded", and other times, "Doing something succeeded" or "doing something succeeded".

Reply:
I think this is fine